From fdafb76e8f243a61e9d771f6da89ae9a1b8a7bc2 Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawnlandden@gmail.com>
Date: Wed, 4 Jul 2012 09:23:16 -0700
Subject: [PATCH 3/6] configure: disable march native, agressive -m64

disable march=native, don't build 64-bit always when -m64 is
 available
---
 configure.in |   22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/configure.in b/configure.in
index 9732529..c11de95 100644
--- a/configure.in
+++ b/configure.in
@@ -211,28 +211,6 @@ test -n "$CXXFLAGS" && MYCXXFLAGS="$CXXFLAGS $MYCXXFLAGS"
 test -n "$CPPFLAGS" && MYCPPFLAGS="$CPPFLAGS $MYCPPFLAGS"
 test -n "$LDFLAGS" && MYLDFLAGS="$LDFLAGS $MYLDFLAGS"
 
-# System-depending optimization
-printf 'checking for 64-bit availability... '
-if printf 'main() {}' | $CC -xc -m64 -o config.tmp - >config.tmp 2>&1
-then
-  MYCFLAGS="-m64 $MYCFLAGS"
-  MYCXXFLAGS="-m64 $MYCXXFLAGS"
-  printf 'yes\n'
-else
-  printf 'no\n'
-fi
-if test "$enable_opt" != "no"
-then
-  printf 'checking for CPU optimization availability... '
-  if printf 'main() {}' | $CC -xc -march=native -o config.tmp - >config.tmp 2>&1
-  then
-    MYCFLAGS="-march=native $MYCFLAGS"
-    MYCXXFLAGS="-march=native $MYCXXFLAGS"
-    printf 'yes\n'
-  else
-    printf 'no\n'
-  fi
-fi
 printf 'checking for useless warnings... '
 if printf 'main() {}' | $CC -xc \
   -Wno-unused-but-set-variable -Wno-unused-but-set-parameter -o config.tmp - >config.tmp 2>&1
-- 
1.7.9.5

